CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 
 🚨 *Breaking Changes*
+* Pickling library switched to `cloudpickle` instead of `dill`.
```

Comment:
**question**: What's the user impact, or why should a user care about this change?

Reply:
Wdyt? 2fa99be